### PR TITLE
Edu benefits history review issue#3053

### LINKED ISSUE #3053: Add EducationHistory to review page
Once the education history page is done, we need to add it to the review page.

### PR DESCRIPTION
**Pull request by alexose**
Closes #3053.

### CODE REVIEW

File: src/js/edu-benefits/components/EducationHistoryReview.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                <td scope="col">City></td>
+                <td scope="col">{period.city.value}</td>
+              </tr>
+              <tr>
+                <td scope="col">State</td>
+                <td scope="col">{period.state.value}</td>
+              </tr>
+              <tr>
+                <td scope="col">From</td>
+                <td scope="col">{period.fromDate.value ? `${period.fromDate.value}/${period.fromDate.value}/${period.fromDate.value}` : null}</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/EducationHistoryReview.jsx]: Missing month/day/year?

File: src/js/edu-benefits/components/EducationHistoryReview.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              </tr>
+              <tr>
+                <td scope="col">To</td>
+                <td scope="col">{period.toDate.value ? `${period.toDate.value}/${period.toDate.value}/${period.toDate.value}` : null}</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/EducationHistoryReview.jsx]: Missing month/day/year?

File: src/js/edu-benefits/components/EducationHistoryReview.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+              </tr>
+              <tr>
+                <td scope="col">Hours</td>
+                <td scope="col">{period.state.hours}</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/EducationHistoryReview.jsx]: Needs `.value`, I think

File: src/js/edu-benefits/components/EducationHistoryReview.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+              </tr>
+              <tr>
+                <td scope="col">Type of hours</td>
+                <td scope="col">{period.state.hoursType}</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/EducationHistoryReview.jsx]: Needs `.value`, I think.

File: src/js/edu-benefits/components/EducationHistoryReview.jsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+              </tr>
+              <tr>
+                <td scope="col">Degree, diploma or certificate received</td>
+                <td scope="col">{period.state.degreeReceived}</td>
```
Comment (jbalboni) [inline src/js/edu-benefits/components/EducationHistoryReview.jsx]: Needs `.value`, I think.

**Comment by jbalboni** [comment]
Once this is merged with master, I think the file location would change and this component needs to be hooked up in routes.jsx.

File: script/build.js [hunk 6]
```diff
@@ -96,6 +96,7 @@ if (options.buildtype === 'production') {
   ignoreList.push('facilities/*');
   ignoreList.push('messaging/*');
   ignoreList.push('rx/*');
+  ignoreList.push('jsonform/*');
```
Comment (jbalboni) [inline script/build.js]: What's this for?

**Review by jbalboni — APPROVED** [review]
LGTM. One change in here that might be unintentional?

File: script/build.js (hunk 6 above)
Comment (alexose) [inline script/build.js]: Whoops, something I was testing out last night.  Deleted!

**Pull request merged**